hoot-js/hoot-js.pro

```diff
@@ -50,13 +50,11 @@ release:OBJECTS_DIR = tmp/release
 else:OBJECTS_DIR = tmp/debug
 DESTDIR = ../lib/
 
-SOURCES += $$files(src/*.cpp, true)
-HEADERS += $$files(src/*.h, true)
+SOURCES += $$files(src/main/*.cpp, true)
+HEADERS += $$files(src/main/*.h, true)
 
 # test files
-SOURCES -= $$files(src/*Test*.cpp, true)
-SOURCES -= $$files(src/*Test*.h, true)
 cppunit {
-SOURCES += $$files(src/*Test*.cpp, true)
-SOURCES += $$files(src/*Test*.h, true)
+SOURCES += $$files(src/test/*.cpp, true)
+SOURCES += $$files(src/test/*.h, true)
```

Comment:
All test files were added twice, once even when `cppunit` isn't found.  That could be a problem in some instances.  Also we don't have to compile the test sources twice.